### PR TITLE
BETA: Register loganpaxton.is-a.dev

### PR DESCRIPTION
**Pull request by loganvxr**
Added `loganpaxton.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: CNAME
Comment (DEV-DIBSTER) [inline CNAME:1]: This file should be deleted.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: CNAME
Comment (LoganPaxton) [reply CNAME:1]: fixed.

Comment (DEV-DIBSTER) [reply CNAME:1]: Has not been fixed, as the file is still here in the PR.

Comment (LoganPaxton) [reply CNAME:1]: Sorry, I didn't hit the "Commit Changes..." button

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**